python-sdk/pachyderm_sdk/api/pps/extension.py

```diff
@@ -148,3 +155,129 @@ def create_secret(
         ).encode()
 
         return super().create_secret(file=file)
+
+    class CreateDatumClient:
```

Comment:
In general it's a bit weird to stack class definitions. It should be fine to have this defined outside the `ApiStub` class. You can prefix an underscore to hint that it's not intended to be public